pom.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0"
-         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xmlns="http://maven.apache.org/POM/4.0.0"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
```

Comment:
Why this change is required?

Reply:
Probably was changed automatically.

Reply:
So you should rollback this changes. Developer who will investigate your code or even this commit could be confused why such change is required, maybe it means something? Or it is related to the task somehow? We should not make changes that do not relate to our task, especially when it is some config file like pom.xml

Reply:
Understood, thank you!